app/src/ui/tutorial/done.tsx

```diff
@@ -25,15 +25,42 @@ interface ITutorialDoneProps {
    * The currently selected repository
    */
   readonly repository: Repository
+
+  /**
+   * If this has not happened, the tuturial completion header will be focused so
+   * that it can be read by screen readers. The purpose of tracking this is so
+   * the focus does not repeatedly get moved to this header if user is navigating
+   * between repositories or history and changes view after completing the tutorial.
+   */
+  readonly tutorialCompletionAnnounced: boolean
+
+  /**
+   * Called when the tutorial completion header has been focused and read by
+   * screen readers
+   */
+  readonly onTutorialCompletionAnnounced: () => void
 }
+
 export class TutorialDone extends React.Component<ITutorialDoneProps, {}> {
+  private header = React.createRef<HTMLHeadingElement>()
+
+  public componentDidMount() {
+    if (this.header.current && !this.props.tutorialCompletionAnnounced) {
+      // Add the header into the tab order so that it can be focused
+      this.header.current.tabIndex = 0
+      this.header.current?.focus()
+      this.props.onTutorialCompletionAnnounced()
+      this.header.current.tabIndex = -1
```

Comment:
This feels a bit hacky with setting the tabIndex, but it is an normal accessibility practice to focus headers at the beginning of content when a user needs redirected. But.. headers shouldn't be in the normal tab order.